handlers/telegram/telegram.go

```diff
@@ -84,16 +84,20 @@ func (h *handler) receiveMessage(ctx context.Context, channel courier.Channel, w
 
 	// deal with attachments
 	mediaURL := ""
+	photoURLs := []string{}
```

Comment:
It's correct, but we can improve a little bit this code line to `var photoURLs []string`, to not have an empty slice declaration using a literal.